### DISCUSSION TITLE
Please tell me how to develop with multiple html files

### DISCUSSION THREAD

**Discussion by takuyandao** [codesandbox/codesandbox-client#6948] [Q&A]
hello everyone! thank you.

If you edit an html file other than index.html and save it with Alt+s, it will return to the index.html page and development will not proceed.

[https://codesandbox.io/s/test-a-tag-h266j7](https://codesandbox.io/s/test-a-tag-h266j7)
(Go to the test.html page, edit the code and save, you will be returned to the index.html page.)

I would appreciate it if you could tell me if there is any workaround!

**Comment by alexnm** [comment]
The static template has an entry file defined as the `main` field in `package.json`. You can only have a single entry file for an app, hence why after each change the preview returns to `index.html`. For working with multiple html files, you normally need a local static server, to give you different html files for different pages. Or, when working locally this just works because the browser loads the html file from your local machine. Unfortunately the preview in codesandbox cannot do that because it relies on that `main` file mentioned above.

**Comment by NupurJariwala** [comment]
how to run second html file in github codespace
